language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/util/TestUtil.java

```diff
@@ -571,13 +601,21 @@ public static String getWorkspaceSymbolResponse(Endpoint serviceEndpoint, String
         return getResponseString(result);
     }
 
+    @Deprecated
```

Comment:
Shall we add the deprecated documentation asking the user to use the new method

Reply:
Done